tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationExpressionNegative.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Shall we call this `AnnotationExpressionCodeAnalysisNegative.java` and similarly the .bal file `annot_attachment_expression_code_analysis_negative.bal`?

Reply:
Done